src/fidesops/models/connectionconfig.py

```diff
@@ -88,3 +92,10 @@ def update_test_status(self, test_status: TestStatus, db: Session) -> None:
         self.last_test_timestamp = datetime.now()
         self.last_test_succeeded = test_status == TestStatus.succeeded
         self.save(db)
+
+    def delete(self, db: Session) -> Optional[Base]:
```

Comment:
This implementation is intentional. Since we still have the object in memory (albeit at this point detached from a DB session) it may still be useful for reference checking attributes that were loaded into memory before the DB record was deleted.

Reply:
got it!